src/Components/Server/src/Circuits/RemoteUriHelper.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.AspNetCore.Components.Server.Circuits
     /// <summary>
     /// A Server-Side Components implementation of <see cref="IUriHelper"/>.
     /// </summary>
-    public class RemoteUriHelper : UriHelperBase
+    internal class RemoteUriHelper : UriHelperBase
```

Comment:
I don't think you can do this because of https://github.com/aspnet/AspNetCore/pull/12260/files#diff-d38af8638441a749771933449b1e1abfL68. I already tried in the past.

Reply:
This is something that we could also avoid routing through JS interop maybe?

Reply:
Ah, you're right. I'll back out that change and we can follow up separately.